schematools/cli.py

```diff
@@ -150,7 +150,8 @@ def permissions_revoke(db_url, role):
 @argument_role
 @argument_scope
 @click.option("--dry-run", is_flag=True, default=False, help="Don't execute the GRANT statements")
-def permissions_apply(db_url, schema_url, profile_url, schema_location, profile_location, role, scope, dry_run):
+@click.option("--create-roles", is_flag=True, default=False, help="Create missing postgres roles")
+def permissions_apply(db_url, schema_url, profile_url, schema_location, profile_location, role, scope, dry_run, create_roles):
     """Set permissions for a postgres role associated with a scope from Amsterdam Schema or Profiles."""
     def _fetch_json(location):
```

Comment:
Je zou deze def kunnen verwijderen en de aanroep naar _fetch_json kunnen aanpassen:

van:
profile = _fetch_json(profile_location)

naar:
profile = schema_fetch_url_file(profile_location)

Deze "schema_fetch_url_file" (die je aantreft in utils.py) doet hetzelfde als de _fetch_json.

Reply:
Goeie, ik heb het veranderd.